logexp/Makefile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+PROJECT ?= google-containers
+IMG = gcr.io/$(PROJECT)/logexp
+TAG = test
+
+.PHONY: build push
+
+all: build
+
+cmd/logexp: cmd/main.go
+	CGO_ENABLED=0 go build -o cmd/logexp k8s.io/test-infra/logexp/cmd
```

Comment:
If you compare with https://github.com/kubernetes/test-infra/blob/master/jenkins/e2e-image/Makefile#L27 you'll notice that we delete the compiled binary after the build step. We do this to reduce the chance of pushing a commit with bad data. Consider copying this pattern?